rfc8826.xml

```diff
@@ -188,31 +215,60 @@ the title) for use on https://www.rfc-editor.org/search. -->
           while scripts are allowed to make HTTP requests via the XMLHttpRequest() API (see <xref target="XmlHttpRequest" format="default"/>)
           those requests are not allowed to be made to any server, but rather solely
           to the same ORIGIN from whence the script came <xref target="RFC6454" format="default"/>
-          (although CORS <xref target="CORS" format="default"/> and WebSockets
+          (although Cross-Origin Resource Sharing (CORS) <xref target="CORS" format="default"/> and WebSockets
           <xref target="RFC6455" format="default"/> provide an escape hatch from this restriction,
-          as described below.)
-          This SAME ORIGIN POLICY (SOP) prevents server A from mounting attacks
+          as described below).
+          This SAME-ORIGIN POLICY (SOP) prevents server A from mounting attacks
           on server B via the user's browser, which protects both the user
-          (e.g., from misuse of his credentials) and the server B (e.g., from
-          DoS attack).
+          (e.g., from misuse of his credentials) and server B (e.g., from
+          DoS attacks).
+
+<!-- [rfced] Section 3.2:  Are "ORIGIN" and "SAME ORIGIN POLICY"
+written in all capitals for emphasis (in which case perhaps we
+could use the <strong> element (Section 2.50 of RFC 7991)), or should
+we write them as "origin" (as used elsewhere in this document and
+this cluster) and "Same-Origin Policy" (as used elsewhere in this
+document and in several published RFCs)?
+
+Also, per the "Gender-Specific Language" section of
+<https://www.rfc-editor.org/styleguide/part2/>, please let us know
+if we may change instances of "his," "him," "himself," and "he" to
+"their," "them," "themselves," and "they."
+
+Original "(as described below.)" has been fixed):
+ For instance,
+ while scripts are allowed to make HTTP requests via the
+ XMLHttpRequest() API (see [XmlHttpRequest]) those requests are not
+ allowed to be made to any server, but rather solely to the same
```

Comment:
So this is not factually correct. There are a number of requests that can go to any server and that roughly matches the capabilities of HTML form elements. This can either be considered part of the same-origin policy or enshrined exceptions to it.

I'm not sure how important those details are for this document however as WebRTC would not use those type of requests anyway (at which point CORS preflights or a TLS handshake become important).

Reply:
Yeah, not sure what I was thinking when I wrote that. Do you have a suggested replacement one-liner?

Reply:
Maybe something like:

> Scripts can make HTTP requests to other servers roughly within the confines of the functionality of HTML forms, but are not able to see responses of those requests unless the server cooperates via CORS. HTTP requests that go beyond HTML forms are preceded by a CORS preflight to ensure the server is prepared to handle them.

(Looking at https://datatracker.ietf.org/doc/draft-ietf-rtcweb-security/?include_text=1 it seems that would require changes in 3.2 and 3.3. You could also look at https://tools.ietf.org/html/rfc6454#section-3.4.2 for inspiration.)

Reply:
@annevk: I tried to shorten this some. PTAL.